src/eddington/fit_functions_list.py

```diff
@@ -248,3 +249,32 @@ def normal(a, x):
     :return: float
     """
     return a[0] * np.exp(-(((x - a[1]) / a[2]) ** 2)) + a[3]
+
+
+@fit_function(
+    n=3,
+    syntax="a[0] * (a[1] ** x) * exp(-a[1]) / gamma(x+1) + a[2]",
```

Comment:
syntax string should use "^" instead of "**", as most users consider it to represent power.